packages/protocol/migrationsConfig.js

```diff
@@ -206,10 +206,6 @@ const DefaultConfig = {
     oracles: [network.from],
     frozen: false,
   },
```

Comment:
any reason to keep the `transferWhitelist` address list configured below (~L573)?

Reply:
I don't remember a reason, so I probably just missed it. It's removed now.